05_syntax_lab/01.py

```diff
@@ -4,10 +4,13 @@
 """
 # Exercise 1
 # By Zvi Figov
-print "Enter 10 positive numbers" # Assuming positive numbers 
+import sys
+
+print "Enter 10 numbers" 
 largest = -1
 for i in range(1,11):
-    print "Enter number %d" % i
+    txtstr =  "Enter number %d : " % i
+    sys.stdout.write(txtstr)
     num = int(raw_input())
     largest = max(num,largest)
 
```

Comment:
I guess I managed to mislead you. Sorry for that.

Yours is a solution but you actually don't need ```sys```, not even ```print``` to show your prompt. Read how ```raw_input``` works. https://docs.python.org/2/library/functions.html#raw_input

Also, please replace the older ```%``` with the newer ```.format``` for creating strings with values in it.